implementación info de todas as clases

desvidrealuis opened the pull request:
Implementación de info das clases fillas é arranxar o getter e setter da clase abstracta

desvidrealuis committed:
implementación info de todas as clases

noevilamunoz commented on Main.cs:
```diff
@@ -48,18 +48,12 @@ public class MainProgram {
             if (turno == 0) { 
-            
-
                 turno = rand.Next(0, 2);
```
Inicializades esta variable antes do `do while` e despóis aquí repetides o mesmo código dentro dos dous if
Podedes facer o seguinte se queredes manter esta estructura e a variable turno:
`do {
    if (turno == 0)  {
        // Do something;
    } else {
        // Do other thing;
    }
`

noevilamunoz replied:
`rand.Next(0,2);` é o mesmo que `rand.Next(2);` sendo este último máis lexible

noevilamunoz commented on Main.cs:
```diff
@@ -69,11 +63,7 @@ public class MainProgram {
-
-
-        // Una vez implementado o bucle de xogo esta li�a mostrara a info das unidades
+        // Una vez implementado o bucle de xogo esta li�a mostrara a info das unidades
```
Neste comentario non sei que cambiáchedes pero o github detecta o "ñ" como un caracter que non coñece

Gontrojon committed:
Arreglos na variable random e no unicode

Optimizouse o funcionamento de asignacion de turnos e fallos cos caracteis unicode
